components/progressbar/stories/progressbar.stories.js

```diff
@@ -1,15 +1,33 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { isIndeterminate, size, staticColor } from "@spectrum-css/preview/types";
 import { ProgressBarGroup } from "./progressbar.test.js";
+import { IndeterminateGroup, Template } from "./template.js";
 
 /**
  * The progress bar component shows the progression of a system operation such as downloading, uploading, processing, etc. in a visual way.
+ * 
+ * ## Values
+ * The value is the progress of a system operation (e.g., downloading, uploading, processing) within the progress bar’s range, from the min value to max value. By default, the min value starts at 0 and the max value is set to 100. These values are not applicable when a progress bar is indeterminate.
+ * 
+ * Progress bars can have a value label at the end of the track that gives detailed information about the progress (e.g. "60%" or "2 of 8"). This value label works alongside the label and should not be displayed if the label itself is not displayed. It should also not be displayed if the progress is indeterminate. Similar to the label, the value label is always placed above the track.
  */
 export default {
 	title: "Progress bar",
 	component: "ProgressBar",
 	argTypes: {
-		customWidth: { table: { disable: true } },
+		customWidth: {
+			name: "Custom width",
+			defaultValue: 192,
+			description: "Adjust the width of the component between 48px and 768px. 192px is the default width on desktop, while 240px is the default on mobile.",
+			type: { name: "number" },
+			table: {
+				type: { summary: "number" },
+				category: "Component",
+				defaultValue: { summary: 192 },
+			},
+			control: { type: "range", min: 48, max: 768,},
+		},
```

Comment:
I brought this over from #2659, but I'm not certain the mobile/desktop scaling is working properly. That would have to be a separate fix in the CSS I think.